dev_vm/roles/devenv/files/startup.patch

```diff
@@ -0,0 +1,18 @@
+diff --git startup.sh startup.sh
+index 9bb4a87..8031977 100755
+--- startup.sh
++++ startup.sh
+@@ -25,9 +25,11 @@ echo '[docker setup]'
+ distro=$(cat /etc/*release | egrep '^ID=' | awk -F= '{print $2}' | tr -d \")
+ echo $distro detected.
+ if [ x"$distro" == x"centos" ]; then
+-   yum install -y docker
++   yum install -y yum-utils device-mapper-persistent-data lvm2
++   yum-config-manager --add-repo https://download.docker.com/linux/centos/docker-ce.repo
++   yum install -y docker-ce
++   echo 'OPTIONS="--storage-opt dm.basesize=20G"' >> /etc/sysconfig/docker
```

Comment:
Shall we increase the VM disk size to 100G? I encountered out-of-disk space issue before

Reply:
Yes, we can. We have a lot of space on the fab servers.

Reply:
The changes look good. Thanks for enhancing this. I will merge it once you increase the size to 100G (or 200G since we have TBs disk space?). 

Let's do a quick TOI on Friday to the team.